upload/catalog/controller/extension/smailyforopencart/cron_customers.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
 /**
  * Samaly subscribers synchronization.
```

Comment:
While we are here. Please fix "Samaly" to "Smaily".

Reply:
Fixed this typo in 0850347